e2e/runner/cypress-runner-run-tests.js

```diff
@@ -2,25 +2,68 @@ const cypress = require("cypress");
 
 const { parseArguments, args } = require("./cypress-runner-utils");
 
-const isOpenMode = args["--open"];
+const DEFAULT_PORT = 4000;
+const getHost = () =>
+  `http://localhost:${process.env.BACKEND_PORT ?? DEFAULT_PORT}`;
 
-const runCypress = async (baseUrl, exitFunction = console.log) => {
-  const defaultConfig = {
-    browser: "chrome",
-    configFile: "e2e/support/cypress.config.js",
-    config: {
-      baseUrl,
-    },
-  };
+const configs = {
```

Comment:
currently there are 3 configs: 
- snapshot
- e2e
- component

but we can add any number of additional configs and make it easy to figure out which you're running by keeping them together here. This could be useful in https://github.com/metabase/metabase/pull/52515